### ISSUE TITLE
Uncaught TypeError: ApolloClient is not a constructor

### ISSUE THREAD

**Issue by bradley-mubenga** [apollographql/apollo-client#10032]
**Intended outcome:**

Hey, community! I am trying to set up my Apollo Client with React v18, I followed the documentation on the Apollo website and I was expecting it to work as usual and to connect to my GraphQL Server but that did not happen, instead, I got the error below. I am getting the same error for the InMemoryCache constructor as well.

**Actual outcome:**

![image](https://user-images.githubusercontent.com/87825432/186378686-4b10955c-2f2a-4a76-9a54-ebe96f46c723.png)
You can follow the steps below to reproduce this problem and maybe find a solution to it. Thanks.

**How to reproduce the issue:**

Install create react app and dependencies.
```bash
npx create-react-app app-name
npm install @apollo/client graphql
```
app-name/index.js
```javascript
import React from 'react';
import { createRoot } from 'react-dom/client';
import App from './App';

//REACT v18+ Renderer
const container = document.getElementById('root');
const root = createRoot(container);
root.render(<App tab="home" />);
```
app-name/app.js
```javascript
import React, { Component } from 'react';
//
import { ApolloClient, InMemoryCache, ApolloProvider, gql } from '@apollo/client';
const client = new ApolloClient({
  uri: 'http://localhost:4000/',
});
export default class App extends Component {
  render() {
    return (
      <div><h1>HELLO!</h1></div>
    )
  }
}
```

**Versions**

  System:
    OS: Linux 5.15 Ubuntu 22.04.1 LTS 22.04.1 LTS (Jammy Jellyfish)
  Browsers:
    Chrome: 104.0.5112.101
  npmPackages:
    @apollo/client: ^3.6.9 => 3.6.9

**Comment by jpvajda** [comment]
Thanks for reporting this! It be helpful to see a reproduction of this issue. You can create a reproduction using https://github.com/apollographql/react-apollo-error-template and link to it here. If you prefer an in-browser way to create reproduction, try: https://codesandbox.io/s/github/apollographql/react-apollo-error-template

**Comment by bradley-mubenga** [comment]
Hey, @jpvajda it turns out that my code works without errors in the online sandbox, I then when ahead and used the package.json to install the same packages and dev dependencies on my local machine and it worked!

I think the issue might be with `create-react-app` or I did not follow the documentation correctly. I will dig deeper into it in my time. 

I am closing this issue since I have found the solution.
